src/components/footer.js

```diff
@@ -0,0 +1,257 @@
+import React from "react"
+import styled from "styled-components"
+import dimensions from "../style/dimensions"
+import { StaticQuery, graphql } from "gatsby"
+
+const FooterContainer = styled.div`
+  background-color: #848484;
+  width: 100%;
+  color: #ffffff;
+`
+
+const FooterMainContent = styled.div`
+  @media (min-width: ${dimensions.maxwidthTablet}px) {
+    display: flex;
+  }
+`
+
+const SocialSection = styled.div`
+  padding-left: 80px;
+  padding-top: 50px;
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    padding-left: 32px;
+  }
+`
+
+const FooterHeader = styled.div`
```

Comment:
If this type style is used in multiple places, can you pull it out into a `typography.js` file please?